gulpfile.js

```diff
@@ -186,13 +190,16 @@ gulp.task('build:dist:min', () => _genDist({minify: true}));
 gulp.task('build:lib:umd', () => _genUmd());
 gulp.task('build:lib:umd:min', () => _genUmd({minify: true}));
 
-gulp.task('build:lib', [
-    'build:lib:esm',
-    'build:lib:cjs',
-    'build:lib:umd',
-    'build:lib:umd:min',
-]);
+gulp.task(
+    'build:lib',
+    gulp.series(
```

Comment:
Oh looks like `gulp@4` introduced `gulp.series`? I'm guessing because there's also a parallel version...

Reply:
Yes I assumed we wanted to use series, but I am open to using parallel.